ui/pages/institutional/manual-connect-custodian/manual-connect-custodian.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useContext } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Text,
+  Box,
+  ButtonIcon,
+  IconName,
+  ButtonIconSize,
+  Button,
+  ButtonVariant,
+  ButtonSize,
+} from '../../../components/component-library';
+
+import {
+  Display,
+  AlignItems,
+  FlexDirection,
+  BlockSize,
+  IconColor,
+} from '../../../helpers/constants/design-system';
+import { mmiActionsFactory } from '../../../store/institutional/institution-background';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import JwtUrlForm from '../../../components/institutional/jwt-url-form';
+import PulseLoader from '../../../components/ui/pulse-loader';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../shared/constants/metametrics';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+
+type ManualConnectCustodianProps = {
+  cancelConnectCustodianToken: () => void;
+  custodianImage: string;
+  custodianDisplayName: string;
+  jwtList: string[];
+  token: string;
+  setCurrentJwt: (jwt: string) => void;
+  loading: boolean;
+  setConnectError: (error: string) => void;
+  custodianName: string;
+  custodianType: string;
+  addNewTokenClicked: boolean;
+  handleConnectError: (e: Error) => void;
+  setAccounts: (accountsValue: object) => void;
+  removeConnectRequest: () => void;
+  connectRequest: object;
+};
+
+const ManualConnectCustodian: React.FC<ManualConnectCustodianProps> = ({
+  custodianImage,
+  custodianDisplayName,
+  jwtList = [],
+  token = '',
+  loading,
+  custodianName,
+  custodianType,
+  addNewTokenClicked,
+  connectRequest,
+  setCurrentJwt,
+  setConnectError,
+  handleConnectError,
+  setAccounts,
+  removeConnectRequest,
+  cancelConnectCustodianToken,
+}) => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const mmiActions = mmiActionsFactory();
+  const trackEvent = useContext(MetaMetricsContext);
+
+  return (
+    <>
+      <Box
+        padding={4}
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        className="page-container__content"
+        width={BlockSize.Full}
+      >
+        {window.innerWidth > 400 && (
+          <Box
+            display={Display.Flex}
+            alignItems={AlignItems.center}
+            marginBottom={4}
+            marginTop={4}
+          >
+            <ButtonIcon
+              data-testid="custody-back-button"
```

Comment:
Nit: if this is not too much trouble, can this be refactored as `manual-connect-custodian-back-button` considering it's rendered in a lower level component?